packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -121,11 +121,19 @@ class BlockListBlock extends Component {
 
 	getBlockWidth( { nativeEvent } ) {
 		const { layout } = nativeEvent;
-		const { blockWidth } = this.state;
+		const { marginHorizontal, blockWidth } = this.props;
+		const spacing = 2 * marginHorizontal;
```

Comment:
Could you move the declaration of `spacing` after the `blockWidth` check? Spacing is not needed if the `blockWidth` is not set.

Reply:
Done